### PR TITLE
Manual exclusion

### PR DESCRIPTION
**Pull request by artloren**
## Describe your changes

Adds EM as optimization target even when the character has 0 EM.

## Issue or discord link

- Fixes [Add manual exclusions for targets to appear in results](https://github.com/frzyc/genshin-optimizer/issues/2156)

## Testing/validation
0 EM,  0 HB%, 0 Phys% Albedo optimization targets list:
manual-exclusion

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [x] I have commented my code in hard-to understand areas.
- [x] I have made corresponding changes to README or wiki.
- [x] For front-end changes, I have updated the corresponding English translations.
- [x] I have run `yarn run mini-ci` locally to validate format and lint.
- [x] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/Components/TargetSelectorModal.tsx [hunk 1]
```diff
@@ -53,12 +53,19 @@ export function TargetSelectorModal({
...
+
                   if (flatOnly && unit === '%') return false
 
                   if (excludeHeal && variant === 'heal') return false
 
-                  if (!showEmptyTargets && node.isEmpty) return false
+                  if (
+                    !showEmptyTargets &&
+                    node.isEmpty &&
+                    (path !== 'eleMas' || key !== 'basic')
```
Comment (nguyentvan7) [inline libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/Components/TargetSelectorModal.tsx:65]: Should be fine to only check if key !== basic. eleMas should already be in there afaik

**Review by nguyentvan7 — CHANGES_REQUESTED** [review]

File: libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/Components/TargetSelectorModal.tsx (hunk 1 above)
Comment (artloren) [reply libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/Components/TargetSelectorModal.tsx:65]: It is already there, but it also shows healing bonus and physical damage bonus. Do you want those to be shown as well?

Comment (nguyentvan7) [reply libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/Components/TargetSelectorModal.tsx:65]: Yes

**Review by nguyentvan7 — APPROVED** [review]
Ty

**Pull request merged**